src/engine/proxy/EngineMethods.h

```diff
@@ -135,8 +138,10 @@ class ENGINE_PROXY_API EngineMethods
     void                     Query(const std::vector<int> &,
                                    const QueryAttributes *,
                                    QueryAttributes &);
-    void                     ExportDatabases(const intVector &ids, const ExportDBAttributes &,
-                                             const std::string &timeSuffix);
+    void                     ExportDatabases(const intVector &ids,
+                                             const ExportDBAttributes *,
+                                             const std::string &timeSuffix,
+                                             ExportDBAttributes &);
```

Comment:
Why anonymous formal arg here? I don't think its necessary but we don't tend to use anonymous formal args except for cases where the arg is a placeholder for future work and isn't actually used in the func.

Reply:
There was already anonymous formal arg. It may be true that we shouldn't use them, but they are used elsewhere in this file.